### DISCUSSION TITLE
Before Submitting my student education, it is redirecting me to github.com/price.

### DISCUSSION THREAD

**Discussion by sajibAdhi** [community/community#103427] [GitHub Education]
### Hi everyone,

Before Submitting my student education, it is redirecting me to github.com/price.

I want to submit my student information on this [application URL](https://education.github.com/discount_requests/application). But before doing anything this application URL is redirecting me to this [pricing URL](https://github.com/pricing).

**Comment by lanbinshijie** [comment]
There are many same problems, but no solutions yet
